DFC.Digital/DFC.Digital.Web.Sitefinity/DFC.Digital.Web.Sitefinity.csproj

```diff
@@ -953,19 +953,20 @@
     <Content Include="AdminApp\assets\manifest.json" />
     <Content Include="OpenAccessNuGet.targets" />
     <Content Include="App_Data\Sitefinity\Export\JobProfile.1.0.25.1.nupkg" />
+    <Content Include="App_Data\Sitefinity\Export\JobProfile.1.0.27.2.nupkg" />
+    <Content Include="Mvc\Views\JobProfileWhatItTakes\DesignerView.Simple.cshtml" />
```

Comment:
Do we need these files under Mvc\Views\JobProfileWhatItTakes in Sitefinity main project?

Reply:
we used to but it seems this later version of Sitefinity allows us to have them co located in the project view folder

Reply:
So this can be deleted as this new addition for WIT is not needed anymore? Am i missing something? This is not related to WYD anyway isn't it?